src/FluentValidation.Tests/ValidatorTesterTester.cs

```diff
@@ -19,25 +19,30 @@
 #endregion
 
 namespace FluentValidation.Tests {
-	using System;
 	using System.Collections.Generic;
 	using System.Linq;
 	using Xunit;
 	using TestHelper;
-	using Validators;
 	using System.Threading.Tasks;
 
 	public class ValidatorTesterTester {
 		private TestValidator validator;
+		private const string sutSameForenameAndSurnameMessage = "A person cannot have the same forename and surname! What the hell, Helle Helle! https://en.wikipedia.org/wiki/Helle_Helle";
 
 		public ValidatorTesterTester() {
 			validator = new TestValidator();
+			validator.RuleFor(x => x).Must(NotHaveSameForenameAndSurname).WithMessage(sutSameForenameAndSurnameMessage);
 			validator.RuleFor(x => x.CreditCard).Must(creditCard => !string.IsNullOrEmpty(creditCard)).WhenAsync((x, cancel) => Task.Run(() => { return x.Age >= 18; }));
 			validator.RuleFor(x => x.Forename).NotNull();
 			validator.RuleForEach(person => person.NickNames).MinimumLength(5);
 			CultureScope.SetDefaultCulture();
 		}
 
+		private bool NotHaveSameForenameAndSurname(Person person)

```

Comment:
Let's get rid of this method and just move it to an inline func as it's only used in the one place